bootstrap4/templatetags/bootstrap4.py

```diff
@@ -315,7 +315,7 @@ def bootstrap_javascript(jquery='falsy'):
     javascript_tags = []
 
     # Set jquery value from setting or leave default.
-    jquery = get_bootstrap_setting('include_jquery', jquery)
+    jquery = get_bootstrap_setting('include_jquery') or jquery
```

Comment:
While have `jquery = None` as default value for function
By my opinion it should be better to use next statement:
`jquery = jquery or get_bootstrap_setting('include_jquery') or 'falsy'`

Reply:
Yeah right, tnx!

Reply:
But this not work, because function https://github.com/METAJIJI/django-bootstrap4/blob/patch-1/bootstrap4/bootstrap.py#L61 return `False` if can't get option!
For example `get_bootstrap_setting('include_jquery', 'default_value')` - return `False` :) lol
because `BOOTSTRAP4_DEFAULTS['include_jquery']` is `False` https://github.com/METAJIJI/django-bootstrap4/blob/patch-1/bootstrap4/bootstrap.py#L39